Index: helpers/__init__.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from genetic_algorithms.helpers.history import History
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/helpers/__init__.py b/helpers/__init__.py
--- a/helpers/__init__.py	
+++ b/helpers/__init__.py	
@@ -1,1 +1,1 @@
-from genetic_algorithms.helpers.history import History
\ No newline at end of file
+from helpers.history import History
\ No newline at end of file
Index: problems/base/decision_variable.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from dataclasses import dataclass\r\nfrom typing import Any\r\n\r\n\r\n@dataclass\r\nclass DecisionVariable:\r\n    \"\"\"\r\n    Value that we are trying to optimize in problem.\r\n    \"\"\"\r\n    idx: int  # TODO: who should be responsible for setting this index ??\r\n    # Probably a ProblemClass, but should it be a generic method or it should a concrete one ??\r\n    value: Any\r\n
===================================================================
diff --git a/problems/base/decision_variable.py b/problems/base/decision_variable.py
--- a/problems/base/decision_variable.py	
+++ b/problems/base/decision_variable.py	
@@ -9,4 +9,4 @@
     """
     idx: int  # TODO: who should be responsible for setting this index ??
     # Probably a ProblemClass, but should it be a generic method or it should a concrete one ??
-    value: Any
+    value: int
